charts/gsp-cluster/charts/gsp-monitoring/values.yaml

```diff
@@ -281,13 +281,6 @@ prometheus-operator:
         jsonData:
           authType: credentials
           defaultRegion: eu-west-2
-    persistence:
-      enabled: true
-      type: pvc
-      accessModes:
-      - ReadWriteOnce
-      size: 10Mi
-      storageClassName: gp2
```

Comment:
why did you move this out of here and into the terraform-generated values.yaml?

For context, my understanding is that this file is the "main" one and the terraform-generated one contains only AWS-specific things.

Reply:
> the terraform-generated one contains only AWS-specific things.

And `gp2` persistent storage is only available in AWS...